.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project or its docs
+title: ''
+labels: kind/feature
```

Comment:
If we have `kind/feature` here we should probably have `kind/bug`?

Reply:
Man I did a _bad_ job at updating those. Yeah I'll update it to `kind/bug`

Reply:
Oh, or to match our `area:` I'll use `kind:`